src/patches/lambda.js

```diff
@@ -39,7 +39,14 @@
       unhandledRejection: (reason, promise) => logWithContext('Unhandled rejection at:', promise, 'reason:', reason)
     };
 
+    let isFinished = false;
+
     const finish = (...args) => {
+      if (isFinished) {
+        logWithContext('The Lambda function called back multiple times. Note that promise resolutions are equivalent to callback invocations.');
```

Comment:
Assuming that this is logged. Will there be enough information to debug the double call? It feels like including the `args` might be helpful. I thought the call stack might be helpful too, but probably less so if it's a Promise resolution.

Reply:
Good idea! I'll add the args to the log message. Thanks!